src/sth_server.js

```diff
@@ -516,36 +516,42 @@
-                  if (isNaN(attributes[j].value)) {
-                    // The attribute value is not a number and consequently not able to be aggregated.
-                    sthLogger.fatal('Attribute value not aggregatable', {
-                      operationType: sthConfig.OPERATION_TYPE.SERVER_LOG
-                    });
-                    continue;
+            if (totalAttributes > 0) {
+              for (var i = 0; i < contextResponses.length; i++) {
+                if (contextResponses[i].contextElement &&
+                  contextResponses[i].contextElement.attributes &&
+                  Array.isArray(contextResponses[i].contextElement.attributes)) {
+                  contextElement = contextResponses[i].contextElement;
+                  attributes = contextElement.attributes;
+                  for (var j = 0; j < attributes.length; j++) {
+                    if (isNaN(attributes[j].value)) {
+                      // The attribute value is not a number and consequently not able to be aggregated.
+                      sthLogger.fatal('Attribute value not aggregatable', {
+                        operationType: sthConfig.OPERATION_TYPE.SERVER_LOG
+                      });
+                      continue;
+                    }
+
+                    attribute = attributes[j];
+
+                    // Compose the collection name for the required data
+                    var databaseName = sthDatabase.getDatabase(
+                      request.headers['fiware-service']);
+
+                    var servicePath = request.headers['fiware-servicepath'];
+
+                    // Store the raw data into the database
+                    storeRawData(databaseName, servicePath, contextElement, attribute, recvTime, counterObj, totalTasks, request, reply);
+
+                    // Store the aggregated data into the database
+                    storeAggregatedData(databaseName, servicePath, contextElement, attribute, recvTime, counterObj, totalTasks, request, reply);
                   }
-
-                  attribute = attributes[j];
-
-                  // Compose the collection name for the required data
-                  var databaseName = sthDatabase.getDatabase(
-                    request.headers['fiware-service']);
-
-                  var servicePath = request.headers['fiware-servicepath'];
-
-                  // Store the raw data into the database
-                  storeRawData(databaseName, servicePath, contextElement, attribute, recvTime, counterObj, totalTasks, request, reply);
-
-                  // Store the aggregated data into the database
-                  storeAggregatedData(databaseName, servicePath, contextElement, attribute, recvTime, counterObj, totalTasks, request, reply);
                 }
               }
+            } else {
+              var error = boom.badRequest('At least one attribute with an aggregatable value should be included in the notification');
```

Comment:
I've been discussing this with @fgalan and he has confirmed that Orion does not care about the Http response, it is the same you send back a `200 OK` or a `400 Bad Request` :) And it makes sense, because in this kind of scenarios Orion is agnostic: if you make a subscription for a certain context data and the notification you receive is not useful for you (either because it is empty, or the values are not aggregatable), that's not Orion business, but yours.